script/bridge213.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I added more context in the commit message 0fd57e53716b084845a5860bad7a5f1cc8607aa6.

This shell script was only used as a way of confirming that the compiler bridge compiles _without_ using sbt. This was needed because sbt 1.0.3 is unable to compile 2.13 code. To clarify, the script is not used as part of the build process to build Zinc. But in the future, if we need to bootstrap again, for 2.13.x changes etc the script might come in handy.

Reply:
Why don't you add it to the `bin` folder and document it saying that it shouldn't be used by normal people, only by you? Also, I'd appreciate if the script is OS cross-compatible.